<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InquiryWindowAccount</class>
 <widget class="QWidget" name="InquiryWindowAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background-image: url(:/Icons/ProgramIcons/bg.png);
}

QToolButton{
font: bold 10pt  &quot;Heebo&quot;;
background-color: rgb(13, 54, 101);
color: rgb(233, 236, 237);
border-radius: 10px;
padding: 5px;
}

QLineEdit {
	font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}

QPushButton{
font: bold 10pt  &quot;Heebo&quot;;
background-color: rgb(13, 54, 101);
color: rgb(233, 236, 237);
border-radius: 10px;
padding: 5px;
}
QPushButton::hover{
border-style: inset;
background-color:rgb(25, 104, 195);
color: rgb(213, 173, 18);

}
QPushButton::pressed{
border: 2px solid rgb(0, 170, 0);
}

QComboBox{
	font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}
QComboBox::drop-down 
{
    border: 0px;
}
QComboBox::down-arrow{
   
	image: url(:/Icons/ProgramIcons/downarrow.png);
    width: 10px;
    height: 10px;
    }

QListView{
font: 9pt &quot;Joan&quot;;
    border: 2px solid gray;
    border-radius: 10px;
	border-color: rgb(13, 54, 101);
     padding:4px 20px 4px 4px;
    background: white;
    selection-background-color: darkgray;
}

QRadioButton::indicator::unchecked{ 
border: 1px solid; 
border-color: rgb(13, 54, 101);
border-radius: 9px;
border-width: 2px;
background-color: white; 
width: 15px; 
height: 15px; 
}

QRadioButton::indicator::checked{ 
border-color: rgb(13, 54, 101);
background-color:  rgb(213, 173, 18);
border: 1px solid; 
border-radius: 9px;
border-width: 2px;
width: 15px; 
height: 15px; 
}

QScrollBar:vertical {
border: none;
	background-color: rgb(208, 244, 255);
width: 14px;
margin: 15px 0 15px 0;
border-radius: 7px;
}


/* HANDLE BAR VERTICAL */
QScrollBar::handle:vertical {
	background-color: rgb(12, 12, 84);
min-height: 30px;
border-radius: 7px;
}
QScrollBar::handle:vertical:hover{
background-color:rgb(28, 114, 206)
}
QScrollBar::handle:vertical:pressed {
	background-color: rgb(8, 32, 59);
}


/* BTN TOP - SCROLLBAR */
QScrollBar::sub-line:vertical {
border: none;
background-color: rgb(213, 173, 18);
height: 15px;
border-top-left-radius: 7px;
border-top-right-radius: 7px;
subcontrol-position: top;
subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical:hover {
background-color: rgb(211, 213, 66);
}
QScrollBar::sub-line:vertical:pressed {
	background-color: rgb(139, 111, 12);
}


/* BTN BOTTOM - SCROLLBAR */
QScrollBar::add-line:vertical {
border: none;
background-color: rgb(213, 173, 18);
height: 15px;
border-bottom-left-radius: 7px;
border-bottom-right-radius: 7px;
subcontrol-position: bottom;
subcontrol-origin: margin;
}
QScrollBar::add-line:vertical:hover {
background-color: rgb(211, 213, 66);
}
QScrollBar::add-line:vertical:pressed {
background-color: rgb(139, 111, 12);
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="leftMargin">
    <number>100</number>
   </property>
   <property name="rightMargin">
    <number>100</number>
   </property>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="15,85">
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="HeaderProfilePicture">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="HeaderFullName">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Full Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="HeaderDepartment">
         <property name="text">
          <string>Department</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="HeaderAccountType">
         <property name="text">
          <string>Account Type</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="AccountDetails">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Your Account Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="EditDetails">
       <property name="text">
        <string>Edit Details</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="UsernameLabel">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="UsernameLineEdit">
         <property name="placeholderText">
          <string>Enter your username</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="OldPasswordLabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="OldPasswordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::PasswordEchoOnEdit</enum>
         </property>
         <property name="placeholderText">
          <string>Enter your old password</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="ChangePassword">
         <property name="text">
          <string>Change Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="MiddleNameLineEdit">
       <property name="placeholderText">
        <string>Enter your middle name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="MiddleNameLabel">
       <property name="text">
        <string>Middle Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="FirstNameLabel">
       <property name="text">
        <string>First Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="FirstNameLineEdit">
       <property name="placeholderText">
        <string>Enter your first name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="LastNameLineEdit">
       <property name="placeholderText">
        <string>Enter your last name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="LastNameLabel">
       <property name="text">
        <string>Last Name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="5" column="0">
          <widget class="QLabel" name="DepartmentLabl">
           <property name="text">
            <string>Department</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="PersonalEmailLabel">
           <property name="text">
            <string>Personal Email</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="6" column="0" colspan="3">
          <widget class="QComboBox" name="DepartmentComboBox"/>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QLineEdit" name="AcademicEmaiLineEdit">
           <property name="placeholderText">
            <string>Enter your academic email</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="3">
          <widget class="QLabel" name="AcademicEmailLabel">
           <property name="text">
            <string>PLM Email</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLineEdit" name="PersonalEmailLineEdit">
           <property name="placeholderText">
            <string>Enter your personal email</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="1" column="1" colspan="2">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="GenderMaleButton">
              <property name="text">
               <string>Male</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="GenderFemaleButton">
              <property name="text">
               <string>Female</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="ContactNumberLineEdit">
           <property name="placeholderText">
            <string>Enter your contact number</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="2">
          <widget class="QLabel" name="GenderLabel">
           <property name="text">
            <string>Gender</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="ContactNumberLabel">
           <property name="text">
            <string>Contact Number</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
        <widget class="QLabel" name="YourTagsLabel">
         <property name="text">
          <string>Your Tags</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="YourTagsList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="YourTagsRemoveButton">
         <property name="text">
          <string>Remove Tag</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="SearchTagsLabel">
         <property name="text">
          <string>Search for Tags</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLineEdit" name="SearchTagsLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="placeholderText">
            <string>Search for tags here</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="SearchTagsList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>175</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
        <widget class="QLabel" name="ProfilePicture">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignBottom">
        <widget class="QLabel" name="ProfilePictureLabel">
         <property name="text">
          <string>Your Profile Picture </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ProfilePictureButton">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ProfilePictureDefaultButton">
         <property name="text">
          <string>Default</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QPushButton" name="SaveButton">
     <property name="text">
      <string>Save Changes</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
